presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -83,6 +83,7 @@ public DockerContainer(String dockerImageName, String logicalName)
     {
         super(dockerImageName);
         this.logicalName = requireNonNull(logicalName, "logicalName is null");
+        this.waitStrategy = null; // require explicitly setting waitStrategy
```

Comment:
Optional? Don't you need to override `waitingForAll` as well?

Reply:
nope. this is defined in parent class. waitingForAll calls waitForAll under the hood

Reply:
i am concerned here that you remove a wait strategy for all containers except the ones we have this configured explicitly. is this a step in a right direction? Perhaps. Is it a safe step?